ckanext/canada/controller.py

```diff
@@ -110,11 +75,52 @@ def org_key(org):
         )
         return render('organization/index.html')
 
+    def datatable(self, resource_id):
+        echo = int(request.params['sEcho'])
+        search_text = unicode(request.params['sSearch'])
+        offset = int(request.params['iDisplayStart'])
+        limit = int(request.params['iDisplayLength'])
+        sort_cols = int(request.params['iSortingCols'])
+        if sort_cols:
+            sort_by_num = int(request.params['iSortCol_0'])
+            sort_order = ('desc' if request.params['sSortDir_0'] == 'desc'
+                          else 'asc')
```

Comment:
I don't do this, and it's not part of my reading of pep8. I find it a waste of vertical space, especially for really simple things